helpers/getFontsCollection.js

```diff
@@ -4,8 +4,8 @@ const getFonts = require('./lib/fonts');
 
 const factory = globals => {
     return function() {
-        const options = arguments.length === 1 ? arguments[arguments.length - 1] : null;
-        const fontDisplay = options && options.hash['font-display'] ? options.hash['font-display'] : null;
+        const options = arguments[arguments.length - 1];
```

Comment:
@bc-williamkwon what is ```arguments``` is empty? that might cause undefined? if so, is the next line might fail?

Reply:
arguments would not be empty because handlebars always generates an options object.

Reply:
@bc-williamkwon Is there documentation on the handlebars options, or a link to where this is generated in the code?

Reply:
this is a pattern implemented in many other helpers:

https://github.com/bigcommerce/paper-handlebars/search?q=arguments%5Barguments.length+-+1%5D&unscoped_q=arguments%5Barguments.length+-+1%5D

Reply:
There is a stack overflow topic that gives more insight into handlebars options object https://stackoverflow.com/questions/24606411/what-is-the-options-parameter-in-a-handlebars-helper-function